app/scripts/lib/multichain-api/wallet-requestPermissions.js

```diff
@@ -1,30 +1,29 @@
+import { pick } from 'lodash';
```

Comment:
Same q: should this really live in the `multichain-api` directory? Feels like it shouldn't imo